### ISSUE TITLE
-- Virus -- on latest version : 1.0.7.185 

### ISSUE THREAD

**Issue by nicolas2k** [XhmikosR/perfmonbar#61]
Hi
I've tried to install this 1.0.7.185 with Avast Antivirus, it's detected a lot of virus during installation and running, I've tried use some exception but it's a lot, it seems no correct !

**Comment by XhmikosR** [comment]
There is no virus... If you insist on  using such programs on your machine
you should first bug the creators of such programs and then everyone else.

My suggestion would be to get rid of any AV software

**Comment by nicolas2k** [comment]
Dear XhmikosR,

It isn't that what I saw with even I had added exceptions and then it fell on another failure, I had a screenshot copy/paste and I stopped: "IDP.Generic" on the file temporary perfmonbar.

First, I'm very very sorry to have disturbed you "the creators of these programs", I simply wanted to notice you of an exceptional case and it would not be necessary to be unpleasant, it is a simple prevention !

And then my suggestion is that I would no longer participate with you and besides it would not be a question to get rid of any AV software (Avast Premium) and on the contrary what you make a big mistake. I think it would be rather than a false positive. 

In past, I have already met on another case I was not the only one and even with other different AV! 

Bye !
